pkgs/by-name/pl/plymouth-matrix-theme/package.nix

```
@@ -0,0 +1,46 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  lib,
+  unstableGitUpdater,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "plymouth-matrix-theme";
+  version = "0.1.0-unstable-19-02-2017";
+  src = fetchFromGitHub {
```

```suggestion
  version = "0.1.0-unstable-19-02-2017";

  src = fetchFromGitHub {
```
Skip a line after version.